docs/working-with-bee/bcrypt.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Bcrypt hashing utility
+id: bcrypt
+---
+
+In order to generate a valid admin password hash you can use any available bcrypt compatible tools, both [online](https://bcrypt-generator.com/) and offline (htpasswd).
+
+For convenience Bee also provides a method to generate and validate password hashes:
+
+```sh
+$ ./bee bcrypt super$ecret
```

Comment:
```suggestion
$ bee bcrypt super$ecret
```

I would expect here that the user has it properly installed with `bee` executable on `PATH`.